epictrack-web/src/components/shared/index.tsx

```diff
@@ -426,10 +426,13 @@ export const GrayBox = ({ children, sx, ...rest }: GrayBoxProps) => {
   );
 };
 
-export const ETPreviewText = styled(Typography)(() => ({
+export const ETPreviewText = styled(Typography)((props) => ({
   fontSize: "14px",
   fontStyle: "normal",
-  fontWeight: "400",
+  fontWeight:
+    props.fontWeight === "bold"

```

Comment:
If it is either bold or regular, then, can we consider a boolean parameter instead of fontWeight. Something like isBold.

Reply:
To accept a custom isBold parameter this component will have to be changed from a styled component and we have to implement it differently, I thought this would be easier. fontWeight is a default style parameter and I just overrode it

Reply:
anyway, changed the implementation to like the other custom topographies and passed bold as a prop